Apsitvarkom.Models/Mapping/CleaningEventProfile.cs

```diff
@@ -22,6 +22,11 @@ private void MapRequests()
         CreateMap<CleaningEventCreateRequest, CleaningEvent>()
             .ForMember(x => x.Id, opt => opt.Ignore())
```

Comment:
here we specify all the properties that we preset - for example progress or spotted in `PollutedLocation`, `Id` here. Let's add the newly pre-set `PollutedLocationId` here as well.

Reply:
since the last commit, am not assigning PollutedLocationId values to cleaningEventDefaults object anymore. Lets the mapper do it's job :) . So it should be now as it is.